eg/Classes/MainFrame/__init__.py

```diff
@@ -521,12 +521,45 @@ def GetPanelDirection(self, panel):
         pos = info.find(";dir=") + 5
         return info[pos]
 
+    def SetWindowStyleFlag(self, *args, **kwargs):
+        """
+        Changes the main frame style depending on the display of the tray icon.
+
+        sets the stlye flags fo the minimize button will be grayed out if the
+        tray icon is shown and there are any open child windows.
+        If the tray icon is not shown the minimize button will function as
+        usual.
+
+        :param args: unused parameter
```

Comment:
nope.. removed the ability for anyone to set the style parameter using the method..
this is intentional. don't want an accidental change. a deliberate one can be done using

  wx.Frame.SetWindowStyleFlag(eg.document.frame, style)

if needed. i don't see where it would be but in the event that it does. we will know it wasn't by accident (in our code) if the call above is done.